src/darwin/Framework/CHIP/MTROTAProviderDelegateBridge.h

```diff
@@ -65,6 +67,7 @@ class MTROTAProviderDelegateBridge : public chip::app::Clusters::OTAProviderDele
     static void ConvertToNotifyUpdateAppliedParams(
         const chip::app::Clusters::OtaSoftwareUpdateProvider::Commands::NotifyUpdateApplied::DecodableType & commandData,
         MTROTASoftwareUpdateProviderClusterNotifyUpdateAppliedParams * commandParams);
+    static void handleBdxInitReceivedTimeoutExpired(chip::System::Layer * systemLayer, void * state);
```

Comment:
This does not seem to be implemented either?

Reply:
its in the .mm file

Reply:
On a different class.